src/components/EditableTicket/EditableTicket.tsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect } from "react";
+import { CSSProperties, FormEvent, KeyboardEvent, useRef, useState } from "react";
+import styled from "styled-components";
+
+import Card from "../Card";
+
+export interface EditableTicketProps {
+  value?: string;
+  placeholder?: string;
+  onBlur?: (value: string) => void;
```

Comment:
It looks like you only use `onBlur` among these three props. Do you think we'll need these properties passed by parents?

Reply:
IMHO, This component is from input component of HTML. And all on... attributes (e.g. onChange, onBlur..) are optional. so I would usually leave them as optional. But I am not sure how other component libraries are doing.